### PR TITLE
ipinfo.io - Added support to use API token for paid plans

### PR DESCRIPTION
**Pull request by omercnet**
fixes https://github.com/demisto/etc/issues/11915

### CODE REVIEW

**Comment by Itay4** [comment]
Did you test it? I assume we don't have an instance.

**Comment by omercnet** [comment]
yes this is tested
API keys are also available free without a plan

File: Integrations/integration-Ipinfo.yml [hunk 1]
```diff
@@ -32,7 +37,7 @@ script:
     }
 
     var base = 'http://ipinfo.io/';
-    var jsonSuffix = '/json';
+    var jsonSuffix = '/json?token=' + params.token;
```
Comment (Itay4) [inline Integrations/integration-Ipinfo.yml]: Should it not be added to `ipinfo_field` command endpoint URL as well?

Comment (omercnet) [reply Integrations/integration-Ipinfo.yml]: yes, fixed

File: Integrations/integration-Ipinfo.yml [hunk 2]
```diff
@@ -50,7 +55,7 @@ script:
             }
             return reply;
         case 'ipinfo_field':
-            return sendRequest(base+args.ip+'/'+args.field, false);
+            return sendRequest(base+args.ip+'/'+args.field+'?token='+args.token, false);
```
Comment (Itay4) [inline Integrations/integration-Ipinfo.yml]: There is no `token` argument, should be `params.token` .
Please create a global `token` variable and use it in both URL addresses.

File: Integrations/integration-Ipinfo.yml [hunk 3]
```diff
@@ -32,7 +37,8 @@ script:
     }
 
     var base = 'http://ipinfo.io/';
-    var jsonSuffix = '/json';
+    var token = params.token;    
+    var jsonSuffix = '/json?token=' + token;
```
Comment (Spivaka) [inline Integrations/integration-Ipinfo.yml]: you will have undefined there when no token

Comment (Spivaka) [reply Integrations/integration-Ipinfo.yml]: you should use ?ternary condition here

File: Integrations/integration-Ipinfo.yml [hunk 4]
```diff
@@ -37,8 +37,8 @@ script:
     }
 
     var base = 'http://ipinfo.io/';
-    var token = params.token;    
-    var jsonSuffix = '/json?token=' + token;
+    var token = params.token ? "?token=" + params.token : "";
```
Comment (Spivaka) [inline Integrations/integration-Ipinfo.yml]: we use ' and not "

**Pull request merged**